test/system/test_helpers.sh

```diff
@@ -114,3 +114,13 @@ create_object() {
         true
     fi
 }
+
+import_object() {
+    local alg_import_obj=0x20040
+
+    if tpm2_import $@ 2>&1 | grep -q 'ERROR: Failed Key Import 000002C2'; then
```

Comment:
If this is an errata of 1.16, can we instead of assuming the TPM2_Import is failing due that, check if the TPM2 revision is 1.16 and CLEAR the sign and decrypt bit? This can be checked with something like following:
```
tpm2_dump_capability -c properties-fixed | grep TPM_PT_REVISION | cut -d ':' -f2 | tr -d '[:space:]'
```

Or extend tpm2_dump_capability to only print the revision.

Reply:
This tool needs a yaml output, that would make this way easier to parse in a programmatic way.

Something that's been building in my head is that each tool outputs a tree of output to tool_main and then tool main can format and output....

Reply:
I implemented a generic framework (https://github.com/01org/tpm2-tools/pull/578) to handle TPM errata. Plz help to review it.